src/dataStrategies.js

```diff
@@ -18,6 +18,7 @@ export const results = ({
   page = 1,
   totalPages = 100,
   include,
+  populate,
```

Comment:
Let's just do a spread and pass along everything else to the results config instead of continually adding these things 1 by 1

Reply:
ok fixed.